spikeinterface/core/waveform_extractor.py

```diff
@@ -118,6 +118,67 @@ def create(cls, recording, sorting, folder, remove_if_exists=False):
 
         return cls(recording, sorting, folder)
 
+    @classmethod
+    def register_extension(cls, extension_class):
+        """
+        This maintain a list of possible extension that are available.
+        This depend on the imported submodule.
```

Comment:
? What do you mean?

Reply:
The list of possible extension is dependant of the import